server/src/config.ts

```diff
@@ -62,8 +62,29 @@ export function getHostInstance(): string {
   }
 }
 
-export function loadDbConfig(): Knex.Config {
+function getPoolSize(connectionName?: string): number {
+  if (!connectionName) return 0;
+
+  const sizeName = `DB_POOL_SIZE_${connectionName.toUpperCase()}`;
+  const rawSize = process.env[sizeName];
+  if (!rawSize) return 0;
+
+  const size = parseInt(rawSize, 10);
+  if (isNaN(size) || size < 1) {
```

Comment:
...but here `0` is invalid and will cause an exception to be thrown.

Reply:
Hmmm, I guess I was thinking:
1. It’s helpful if this stops you from doing something funky (did you mean to never make any connections? to use the default [then why not specify nothing]? did you make a typo? did we have a parsing problem?).
2. But internally the contract is clearer, and it’s easy enough for `0` to be a non-value for a number-returning function.

But I take your point, it’s probably not that important to protect against 0, and it would be overall more consistent.

Reply:
Updated to accept 0.